discord-service/src/main/java/pl/flyingoctopus/discord/command/LinksCommand.java

```diff
@@ -0,0 +1,28 @@
+package pl.flyingoctopus.discord.command;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import pl.flyingoctopus.discord.action.DiscordAction;
+import pl.flyingoctopus.discord.action.HelpAction;
+import pl.flyingoctopus.discord.action.LinksAction;
+
+import java.util.Set;
+import java.util.regex.Pattern;
+
+@Component
+@RequiredArgsConstructor
+public class LinksCommand implements DiscordCommand {
```

Comment:
W zasadzie to chyba nie potrzebna klasa, może od razu LinksAction być używane. Tylko dodać tam helpa

Reply:
Czy help w przypadku LinksAction jest w ogóle potrzebne? Nie ma żadnych dodatkowych opcji w tej komendzie 🤔

Reply:
Hmm, tak bo z tych helpów sobie dokumentację zrobimy. I jak będziemy robić generalnego helpa do wszystkiego to z tego też będziemy korzystać zamiast pisać na nowo.